raiden/network/discovery.py

```diff
@@ -45,7 +45,8 @@ class ContractDiscovery(Discovery):
     """On chain smart contract raiden node discovery: allows to register endpoints (host, port) for
     your ethereum-/raiden-address and looking up endpoints for other ethereum-/raiden-addressess.
     """
-    def __init__(self, blockchainservice, discovery_contract_address, poll_timeout=DEFAULT_POLL_TIMEOUT):
+    def __init__(self, blockchainservice, discovery_contract_address,
```

Comment:
This is a matter of style that we have discussed before. If the line breaks on a function's arguments then please format it like:

```python
def __init__(
    self,
    blockchainservice,
    discovery_contract_address,
    poll_timeout=DEFAULT_POLL_TIMEOUT
):
```

Why? It's actually quite simple. First it looks symmetric, simple and organized. But for a more practical reason, when refactoring the code later and adding/removing arguments you just need to change one entry and don't have to mess with rearranging the entire arguments line.